docs/users.md

```diff
@@ -522,6 +600,66 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to adopt
+a kind cluster named `ext1`. This setup requires that the external cluster `ext1` 
+and the KubeFlex hosting cluster are on the same docker network.
```

Comment:
"This setup" is not entirely clear. Is it saying that adoption can only be done with both clusters are on the same Docker network (I hope this is not true), or is it saying that this _example_ involves two clusters on the same Docker network? If the latter then I would reword to say something like "This example supposes that ...".

Reply:
Updated as suggested.